INSTALL.rst.txt

```diff
@@ -20,7 +20,7 @@ Building NumPy requires the following installed software:
    e.g., on Debian/Ubuntu one needs to install both `python3` and
    `python3-dev`. On Windows and macOS this is normally not an issue.
 
-2) Cython >= 0.29.14
+2) Cython >= 0.29.21
```

Comment:
Do we care about supporting older cython versions in older python versions?

Reply:
@eric-wieser That sounds too complicated and we are still on the same Cython minor version, only the patch version has changed. Maintaining compatibility in this case is on Cython. Something might slip through the cracks but I expect it would be discovered eventually and reported upstream.